### DISCUSSION TITLE
Do you need to define a store when using Pinia ORM?

### DISCUSSION THREAD

**Discussion by pstaal** [CodeDredd/pinia-orm#1964] [General]
Since using the model already initiates a Pinia store, I was wondering if we still need a call to defineStore somewhere else? To me this seems to to be redundant, but maybe I am missing something. I do find examples that use a model and a call to defineStore; how does that work? Do they overwrite each other?

**Comment by CodeDredd** [comment]
You don't need a `defineStore`. If you create/write a model then this will call internally `defineStore` creating a store based on the model/repository.
They don't overwrite each other. You still can use seperate pinia stores.
